### DISCUSSION TITLE
Thermostat card for water_heater entity

### DISCUSSION THREAD

**Discussion by chomupashchuk** [home-assistant/frontend#11249] [Other feature requests] [answered]
## The request
Allow use of thermostat card for water_heater entity, there is no proper card for water_heater yet.

## The alternatives
New card

**Comment by zsarnett** [comment]
Would you be able to post the states and attributes your entity has?

**Comment by chomupashchuk** [comment]
State:
Winter [taken from operation list]

Attributes:
min_temp: 36
max_temp: 60
operation_list: Off, Summer, Winter
current_temperature: null
temperature: 36
target_temp_high: null
target_temp_low: null
operation_mode: Winter
target_temp_step: 1
friendly_name: Ariston
icon: mdi:water-pump
supported_features: 3

**Comment by chomupashchuk** [comment]
As i understand water_heater is a fork of climate entity. Water heater can have temperature range, setting of the temperature and can support current temperature reading. So from frontend would be nice to show at least temperature slider like in climate card with current operation being shown.

**Comment by zsarnett** [comment]
This should be just adding the ability to set the water_heater domain to the card and to the editor

**Comment by karwosts** [comment, marked as answer]
Added for 2025.10.
